management/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@fewlines/connect-management",
-  "version": "0.6.6",
+  "version": "0.6.7",
```

According to my previous comment
```suggestion
  "version": "0.7.0",
```